### ISSUE TITLE
Incorrect fee calculation when using `SignatureAlgorithm.ECDSA`

### ISSUE THREAD

**Issue by khirvy019** [CashScript/cashscript#220]
When using `SignatureAlgorithm.ECDSA` in `SignatureTemplate` as a parameter in a smart contract, the resulting signature length is 72 bytes.

However when calculating the fees internally the transaction it replaces a placeholder of 65 bytes length only [here](https://github.com/CashScript/cashscript/blob/0dd95d5b05a438ac1884bdb2252aec6a4c77e5b2/packages/cashscript/src/Transaction.ts#L339)
```
// Replace all SignatureTemplate with 65-length placeholder Uint8Arrays
const placeholderArgs = this.encodedFunctionArgs.map((arg) => (
    arg instanceof SignatureTemplate ? placeholder(65) : arg
));
```

This causes a `min relay fee` error when validating the resulting transaction since the change output to be off by 6 bytes for each signature parameter

Would it be possible to check the signature algorithm to determine the placeholder length instead of using a fixed 65bytes?

**Comment by rkalis** [comment]
Hey @khirvy019 thanks for the report. We've implemented a fix which will be released next week Monday in our 0.10.2 release. So once that is live next week, you should be able to send transactions with ECDSA signatures. But we recommend using Schnorr any way if you don't have a strong reason not to.

**Comment by khirvy019** [comment]
Thanks @rkalis , we are use it for `checkMultiSig`. We could use Schnorr if there is a way to use `OP_CHECKMULTISIG` mode2.

**Comment by rkalis** [comment]
Hey @khirvy019, we just released v0.10.2, which includes the fix.
